test/assertInterpolation.js

```diff
@@ -0,0 +1,64 @@
+/* global assert suite test */
+
+function assertInterpolation (transformation, expectation) {
+  var target = document.createElement('div');
+
+  for (var {at, is} of expectation) {
+    var timing = {duration: 1, fill: 'forwards'};
+    var currentTime = at;
```

Comment:
No need for currentTime variable.

Add assertions for which values are acceptable for at.